app/controllers/items_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ItemsController < ApplicationController
+
+  def new
+      @user = User.find(params[:user_id])
+      @item = Item.new
+  end
+
+
+  def create
+      @user = User.find(params[:user_id])
+      @items = @user.items
+
+      @item = Item.new  
+      @item = current_user
```

Comment:
Why are we reseting @item to current_user. Did you mean `@item.user = current_user`?